packages/main/test/pages/Carousel.html

```diff
@@ -2,434 +2,496 @@
-        <ui5-button>Button 3</ui5-button>
-        <ui5-button>Button 4</ui5-button>
-        <ui5-button>Button 5</ui5-button>
-        <ui5-button>Button 6</ui5-button>
-        <ui5-button>Button 7</ui5-button>
-        <ui5-button>Button 8</ui5-button>
-    </ui5-carousel>
-
-    <ui5-title>Carousel with one page</ui5-title>
-    <ui5-label>The arrows and dots are not displayed</ui5-label>
-
-    <ui5-carousel id="carousel6" items-per-page-m="3" items-per-page-l="4">
-        <ui5-button>Button 1</ui5-button>
-        <ui5-button>Button 2</ui5-button>
-        <ui5-button>Button 3</ui5-button>
-    </ui5-carousel>
-
-    <ui5-carousel id="carousel8" items-per-page-m="4" items-per-page-l="4">
-        <ui5-button>Button 1</ui5-button>
-        <ui5-button>Button 2</ui5-button>
-        <ui5-button>Button 3</ui5-button>
-        <ui5-button>Button 4</ui5-button>
-        <ui5-button>Button 5</ui5-button>
-        <ui5-button>Button 6</ui5-button>
-        <ui5-button>Button 7</ui5-button>
-        <ui5-button>Button 8</ui5-button>
-    </ui5-carousel>
-    <ui5-input id="result"></ui5-input>
-    <ui5-input id="resultCounter"></ui5-input>
-
-    <ui5-carousel id="carousel9" items-per-page-m="2">
-        <ui5-button>Button 1</ui5-button>
-        <ui5-button>Button 2</ui5-button>
-        <ui5-button>Button 3</ui5-button>
-        <ui5-button>Button 4</ui5-button>
-        <ui5-button>Button 5</ui5-button>
-        <ui5-button>Button 6</ui5-button>
-        <ui5-button>Button 7</ui5-button>
-        <ui5-button>Button 8</ui5-button>
-        <ui5-button>Button 9</ui5-button>
-        <ui5-button>Button 10</ui5-button>
-    </ui5-carousel>
-    <ui5-input id="visible-items" value="0"></ui5-input>
-
-    <ui5-title style="margin-top: 2rem;">Carousel with hidden page indicator</ui5-title>
-    <ui5-label>arrows-placement="Navigation"</ui5-label>
-    <ui5-carousel id="carouselHiddenPageIndicator" arrows-placement="Navigation" hide-page-indicator>
-        <ui5-button>Button 1</ui5-button>
-        <ui5-button>Button 2</ui5-button>
-        <ui5-button>Button 3</ui5-button>
-    </ui5-carousel>
-
-    <ui5-title style="margin-top: 2rem;">Carousel with hidden page indicator and hidden navigation arrows</ui5-title>
-    <ui5-label>arrows-placement="Navigation"</ui5-label>
-    <ui5-carousel id="carouselHiddenPageIndicatorHiddenArrows" arrows-placement="Navigation" hide-page-indicator hide-navigation-arrows>
-        <ui5-button>Button 1</ui5-button>
-        <ui5-button>Button 2</ui5-button>
-        <ui5-button>Button 3</ui5-button>
-    </ui5-carousel>
+	<style>
+		ui5-carousel {
+			height: 25rem;
+			margin-bottom: 1rem;
+		}
+
+		.myCard {
+			height: 100%;
+		}
+	</style>
+
+	<ui5-carousel id="carouselCards" items-per-page-s="3" items-per-page-m="3" items-per-page-l="3">
+		<ui5-card id="card"
+			class="myCard">
+			<ui5-card-header status="1 of 11" title-text="Quick Links" subtitle-text="quick links" header-interactive>
```

Comment:
Cant see the slot="header" here as well. You can check all html files that are changed.